src/python/epicscorelibs/test/test_ioc_ca.py

```diff
@@ -0,0 +1,97 @@
+import subprocess
+import os
+import sys
+import ctypes
+import tempfile
+import random
+import string
+
+try:
+    from queue import Queue
+except ImportError:
+    from Queue import Queue
+
+from epicscorelibs.ca import cadef, dbr
+
+
+class Channel:
+    def __init__(self, name):
+        self.name = name
+        self.chid = ctypes.c_void_p()
+
+    @property
+    def _as_parameter_(self):
+        # Setting this allows a channel object to autoconvert into the chid
+        # when passed to ca_ functions.
+        return self.chid.value
+
+
+def test_running_ioc():
+    if os.environ.get("PYTAG", "") == "win_amd64" \
+            and sys.version_info[:2] == (3, 4):
+        # This particular combination is broken. All the callbacks seem to
+        # return junk. Not sure why
+        return
+
+    # Get something reasonably unique for the PV prefix
+    pv = ''.join(random.choice(string.ascii_uppercase) for _ in range(12))
+    dbfile = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
imo `delete=False` negates half the reason for using `NamedTemporaryFile`.  I suspect that you found that doing otherwise breaks on Windows due to the use of `O_TEMPORARY`.

I prefer test code to cleanup after itself.  Either something like is done in [p4p](https://github.com/mdavidsaver/p4p/blob/1a662689661a2ae2b85179017d7f415c9e705bb2/src/p4p/test/utils.py#L175-L193), or as a separate data file in the `test/` directory.

Reply:
I tried putting it into a separate data file. Then completely failed to get it into the built wheel. Now I have just remembered about the MANIFEST.in file. Oh well. I put it inside the try/finally to make it clean up after itself a bit better...

Reply:
try/finally is good.

fyi. MANIFEST.in would only get it into the source tar.  `setup(package_data=...)` would get it into the binary wheel.